apps/admin-x-activitypub/src/components/modals/NewNoteModal.tsx

```
@@ -65,8 +65,23 @@ const NewNoteModal: React.FC<NewNoteModalProps> = ({children, ...props}) => {
+            let errorMessage = 'Failed to upload image. Try again.';
+
+            if (error && typeof error === 'object' && 'statusCode' in error) {
+                switch (error.statusCode) {
+                case 413:
+                    errorMessage = 'Image size exceeds limit.';
+                    break;
+                case 415:
+                    errorMessage = 'The file type is not supported.';
+                    break;
+                default:
+                    errorMessage = errorMessage;
```

Biome (1.9.4)

[error] errorMessage is assigned to itself.

This is where is assigned.


(lint/correctness/noSelfAssign)